SDK/bundles/org.eclipse.embedcdt.debug.gdbjtag.openocd.core/src/org/eclipse/embedcdt/debug/gdbjtag/openocd/core/preferences/DefaultPreferences.java

```diff
@@ -73,6 +73,8 @@ public class DefaultPreferences extends org.eclipse.embedcdt.debug.gdbjtag.core.
 
 	public static final boolean DO_CONTINUE_DEFAULT = true;
 
+	public static final String DEFAULT_REGISTER_FILE = "${eclipse_home}/../registers/csr/gdb_reg_riscv_rv32.xml";
```

Comment:
This can cause issues, I think. By default, we don't need to pass any register files. What if user doesn't browse any register files and the connected target is 64?

Reply:
default file removed